jina/serve/runtimes/worker/__init__.py

```diff
@@ -32,7 +33,7 @@ def __init__(
         :param args: args from CLI
         :param kwargs: keyword args
         """
-        self._health_servicer = health.aio.HealthServicer()
+        self._health_servicer = health.HealthServicer(experimental_thread_pool=ThreadPoolExecutor(1))
```

Comment:
Removing the `aio.HealthServicer` will break the docker + OpenTelemetry fix that I made earlier this week. If we must use the regular `HealthServicer`, we need to adapt the tracing server interceptor to support both any type of rpc.

Reply:
what was the reason of this? can u make a proposal of fix to this branch?

Reply:
I will create a separate pr to fix the tracing server interceptor.

Reply:
The [PR](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1245) for grpc aio support has been merged and released so we can remove our internal dependency in another pr.